### DISCUSSION TITLE
Delete stock from a part with serial number stock

### DISCUSSION THREAD

**Discussion by chechigp** [inventree/InvenTree#5074] [General]
Maybe is a dumb question but I'm trying to start using Inventree and i'm struggling with this.
I have the need to track with a serial number most of the parts we use, sometimes this parts are samples that are used in a prototipe or are damaged by any reason. So I need to remove it from the stock but without loosing the serial number (that serial number can't be used by other stock part).
When I tried to remove the stock the system say: "You must select at least one available stock item". And the only way to delete it is using the "Delete stock" function but as the stock serial number are incremental I saw that this liberate the this serial number.
What I'm doing it wrong?

Thanks for your time

**Comment by SchrodingersGat** [comment]
> Sometimes this parts are samples that are used in a prototipe or are damaged by any reason.

You should use the [stock status field](https://docs.inventree.org/en/latest/stock/status/) to mark the item as unavailable. This way, the serial numbers will still be in the system, but those items will be removed from use.

**Comment by chechigp** [comment]
Thanks a lot, that work as expected.

Thanks!
